src/libraries/Microsoft.PowerFx.Core/Types/TypeResolverResult.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Microsoft.PowerFx.Core.Errors;
+using Microsoft.PowerFx.Core.Parser;
+using Microsoft.PowerFx.Core.Utils;
+using Microsoft.PowerFx.Syntax;
+using Microsoft.PowerFx.Types;
+
+namespace Microsoft.PowerFx.Core.Types
+{
+    internal class TypeResolverResult
```

Comment:
>TypeResolverResult [](http://example.com/codeflow?start=19&length=18)

Can we make this use public  types? 

Does this become like the moral equivalent of CheckResult?

Reply:
Public error class: 
[https://github.com/microsoft/Power-Fx/blob/c7a5af75ceb060a6585173ede1f1afcb2171c08a/src/libraries/Microsoft.PowerFx.Core/Public/CheckResult.cs#L503](https://github.com/microsoft/Power-Fx/blob/c7a5af75ceb060a6585173ede1f1afcb2171c08a/src/libraries/Microsoft.PowerFx.Core/Public/CheckResult.cs#L503)